### ISSUE TITLE
delete stubs if the lib ends up being flow typed

### ISSUE THREAD

**Issue by Brianzchen** [flow-typed/flow-typed#4158]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
If I install a dependency and it's not flowtyped currently we will install a stub. But if the lib becomes flow typed in the future as I'm doing right now across some libs in my team it doesn't delete the file so there needs to be manual intervention

**If the current behavior is a bug, please provide the steps to reproduce.**
Pretty self explanatory I hope

**What is the expected behavior?**
If I run `flow-typed install` and it skips because there is a typed lib, then also search if there's a stubbed module for that lib and if so delete it

**If this is a feature request, what is motivation or use case for changing the behavior?**
It gives a cleaner DX and encourages more flowtype users

**Comment by pascalduez** [comment]
Good idea, but removing the stub automatically is a bit risky, let's say the user wrote some custom defs for their needs without contributing it to flow-typed. An interactive way with a prompt would be cool, obviously more work...

**Comment by Brianzchen** [comment]
We already have logic that removes stubs after a real type def is defined, but we stop if the stub has been modified. I think we could lean on the same logic there and we'll be safe

**Comment by Brianzchen** [comment]
fyi message looks like this
```
universal-cookie@^4.0.3': Stub already exists and has been modified. Use --overwrite to overwrite
```
